### PR TITLE
Remove optional metadata query parameter

### PR DESCRIPTION
**Pull request by ischasny**
Remove metadata query parameter that changed behaviour of whether metadata has been included into extended provider results. Including metadata is required so that the client can correctly assemble extended provider results.

### CODE REVIEW

**Review by gammazero — CHANGES_REQUESTED** [review]
Also need to remove parameter from clients:
- https://github.com/ipni/storetheindex/blob/main/api/v0/finder/client/http/client.go#L74
- https://github.com/ipni/storetheindex/blob/main/api/v0/finder/client/libp2p/client.go#L98-L107
- https://github.com/ipni/storetheindex/blob/main/api/v0/finder/client/interface.go#L23
- https://github.com/ipni/storetheindex/blob/main/command/providers.go#L62

**Comment by ischasny** [comment]
@gammazero good catch, thanks. Removed - please re-review.

File: api/v0/finder/client/interface.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ type Finder interface {
 	// ListPrividers gets information about all providers known to the indexer.
 	// The withExtMetadata requests that extended provider metedata be
 	// included in the response.
```
Comment (gammazero) [inline api/v0/finder/client/interface.go:22]:
```suggestion
```

**Review by gammazero — APPROVED** [review]
One suggestion.

**Pull request merged**